src/Custom/Chat/Internal/InternalChatCompletionRequestMessageContentPartImageImageUrl.cs

```diff
@@ -8,6 +8,8 @@ namespace OpenAI.Chat;
 [CodeGenSuppress("InternalChatCompletionRequestMessageContentPartImageImageUrl", typeof(string))]
 internal partial class InternalChatCompletionRequestMessageContentPartImageImageUrl
 {
+    private static readonly Regex s_parseDataUriRegex = new(@"^data:(?<type>.+?);base64,(?<data>.+)$", RegexOptions.Compiled);
```

Comment:
Note that when the library multitargets .NET 8, this should be updated to use the regex source generator.